expo/gbe/models/show_vote.py

```diff
@@ -0,0 +1,26 @@
+from django.db.models import (
+    ForeignKey,
+    Model,
+    IntegerField,
+)
+
+from gbe.models import (
+    Act,
+    Show,
+)
+from gbetext import vote_options
+
+
+class ShowVote(Model):
+
+    show = ForeignKey(Show,
+                      blank=True,
+                      null=True)
+    vote = IntegerField(choices=vote_options,
+                        blank=True,
+                        null=True)
+
+    class Meta:
+        verbose_name = "show vote"
+        verbose_name_plural = "show votes"
+        app_label = "gbe"
```

Comment:
A nice to have would be a "_str_" function that gives some detail on what this is.  Right now the default is just the class name.  Given how little data is here, I'd make it the output be something like vote-str(show) - even if the vote is just an integer.  It's mostly for admin type stuff.